index.html

```diff
@@ -1835,17 +1835,25 @@ <h2>Forms</h2>
                     <label class="form-control-label" for="input1">Normal input</label>
                     <input type="text" class="form-control" id="input1">
                 </div>
-                <div class="form-group ps-number" data-max="300" data-min="0" data-label-max="Maximum :" data-label-min="Minimum :" data-label-nan="Not a number.">
+                <div class="form-group ps-number-input" data-max="300" data-min="0" data-label-max="Maximum:" data-label-min="Minimum:" data-label-nan="Not a number.">
```

Comment:
Isn't changing `ps-number` to `ps-number-input` a breaking change?

Reply:
Its a component introduced in the next release, it doesn't exist yet anywhere !

https://github.com/PrestaShop/prestashop-ui-kit/pull/86/files